python/semantic_kernel/connectors/ai/open_ai/models/chat/azure_chat_with_data_settings.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) Microsoft. All rights reserved.
+"""Classes to handle Azure OpenAI Chat With Data settings."""
+from dataclasses import dataclass, field
+from enum import Enum
+from typing import Any, Dict, Optional
+
+
+class DataSourceType(Enum):
+    """Enum to hold data source type."""
+
+    AZURE_AI_SEARCH = "AzureCognitiveSearch"
+
+
+@dataclass
+class DataSourceParametersBase:
```

Comment:
Why is this base class empty? then what does it add? I prefer not to add classes that are only there because in the future there might be others, unless you already add a second subclass.

Reply:
There will be others in the future - actually the AOAI with data API already supports several others, but they are in preview and lower priority than AI search for this change. I can remove the base class for now.